support-frontend/assets/helpers/subscriptionsForms/submit.js

```diff
@@ -80,42 +76,40 @@ function getAddresses(state: AnyCheckoutState) {
   };
 }
 
-const getOptions = (
-  fulfilmentOptions: FulfilmentOptions,
-  productOptions: ProductOptions,
-): {fulfilmentOptions: FulfilmentOptions, productOptions: ProductOptions} =>
-  ({
-    ...(fulfilmentOptions !== NoFulfilmentOptions ? { fulfilmentOptions } : {}),
-    ...(productOptions !== NoProductOptions ? { productOptions } : {}),
-  });
-
 const getProduct =
-  (state: AnyCheckoutState, currencyId?: Option<IsoCurrency>): DigitalSubscription | PaperSubscription => {
+  (state: AnyCheckoutState, currencyId?: Option<IsoCurrency>): SubscriptionProductFields => {
     const {
       billingPeriod,
       fulfilmentOption,
       productOption,
       orderIsAGift,
       product,
```

Comment:
rename incoming `product` param to `productType`

Reply:
is this not a field that already exists in the state?

Reply:
Ah yes sorry thought it was a param